app/bin/run-ecs-task/README.md

```diff
@@ -0,0 +1,62 @@
+# Run Task
+The `run-task.sh` is a wrapper shell script around the AWS run task command for starting ECS tasks. It helps setup and find the parameters necessary for starting an ECS task.
+
+# Setup
+This command requires you to have the [AWS CLI](https://aws.amazon.com/cli/) setup and configured locally.
+
+It also heavily uses `jq` for parsing and generating the CSV configurations.
+
+# Usage
+
+`./run-task.sh -a your.name -e test -t your-task-definition` is the most basic way to run a task.
+
+The script has the following parameters:
+* `-a` - The author running the script (eg. if your name is Bob Smith, put `-a bob.smith`)
+* `-e` - The environment you want to run the script in (eg. `-e test`)
```

Comment:
Clarifying question: This should match the environment as named in terraform?

Reply:
~Yes. It currently is only used to specify the ECS cluster to run in (which is named exactly the environment like `test`). If we had a bit more setup, we'd likely use it for a few other pieces like the name of the container. Actually, let me just change that so it's not hardcoded to `test-mock-api-container` but instead does `{env}-mock-api-container`